spec/registrant_spec.rb

```diff
@@ -0,0 +1,124 @@
+require 'spec_helper'
+
+RSpec.describe Registrant do
+  describe "Iteration 1" do
+    it "exists" do
+      registrant_1 = Registrant.new('Bruce', 18, true )
+      registrant_2 = Registrant.new('Penny', 15 )
+
+      expect(registrant_1).to be_a(Registrant)
+      expect(registrant_2).to be_a(Registrant)
+    end
+    
+    it "has attributes" do
+      registrant_1 = Registrant.new('Bruce', 18, true )
+      registrant_2 = Registrant.new('Penny', 15 )
+
+      expect(registrant_1.name).to eq('Bruce')
+      expect(registrant_2.name).to eq('Penny')
+      expect(registrant_1.age).to eq(18)
+      expect(registrant_2.age).to eq(15)
+      expect(registrant_1.permit?).to eq(true)
+      expect(registrant_2.permit?).to eq(false)
+    end
+    
+    it "has list of license data" do
+      registrant_1 = Registrant.new('Bruce', 18, true )
+      registrant_2 = Registrant.new('Penny', 15 )
+
+      expect(registrant_1.license_data).to eq({:written=>false, :license=>false, :renewed=>false})
+      expect(registrant_2.license_data).to eq({:written=>false, :license=>false, :renewed=>false})
+    end
+    
+    it "can earn_permit" do
+      registrant_1 = Registrant.new('Bruce', 18, true )
+      registrant_2 = Registrant.new('Penny', 15 )
+
+      registrant_2.earn_permit
+
```

Comment:
You can make this even more robust by possibly creating another `registrant` who's age is lower than 15 and trying to allow them to `earn_permit` to really check that your method is doing what you want.